### DISCUSSION TITLE
Expo app not opening on phone

### DISCUSSION THREAD

**Discussion by Vuyisile619** [expo/expo#16101] [Archived: Questions]
I have cleared my cache and uninstalled and reinstalled the expo go app but it will not open. It just freezes on the logo part when opening and just stays there. It doen't get to the part where it shows the list of projects so what could be the problem. How can I solve this?.

**Comment by bnussman** [comment]
Same issue on SDK 44 for me

**Reply by Vuyisile619** [reply]
Are you still having the problem?. Mine is still persisting
